fe-nc-news-react/src/components/Comments.jsx

```diff
@@ -1,21 +1,65 @@
 import { useEffect, useState } from "react";
 import { fetchCommentsbyArticleId } from "../api";
 import CommentCard from "./CommentCard";
+import Textarea from '@mui/joy/Textarea';
+import { postNewComment } from "../api";
 
 export default function Comments({ article_id }) {
-    const [comments, setComments] = useState([])
+  const [comments, setComments] = useState([])
+  const [newComment, setNewComment] = useState({});
+  const [newInput, setNewInput] = useState("")
+  const [pendingComment, setPendingComment] = useState(false)
+  const [commentPosted, setCommentPosted] = useState(false)
+  const [isError, setIsError] = useState(false)
+
     useEffect(() => {
         fetchCommentsbyArticleId(article_id)
             .then((comments) => {
             setComments(comments.data.comments);
           })
           .catch()
     }, [article_id])
+    
+  function handleCommentPost(e) {
+    e.preventDefault();
+    setNewComment({ username: 'cooljmessy', body: newInput })
+    setNewInput("")
+  }
+
+  useEffect(() => {
```

Comment:
You don't need a useEffect to post a comment it's an event that can be triggered by pressing a button